Cesium.CodeGen/Ir/Types/CTypeSystem.cs

```diff
@@ -0,0 +1,11 @@
+using Mono.Cecil;
+
+namespace Cesium.CodeGen.Ir.Types;
+
+internal class CTypeSystem
+{
+    public IType Boolean { get; } = new PrimitiveType(PrimitiveTypeKind.Int);
+    public IType Char { get; } = new PrimitiveType(PrimitiveTypeKind.Char);
+    public IType Int { get; } = new PrimitiveType(PrimitiveTypeKind.Int);
+    public IType CharPtr { get; } = new PointerType(new PrimitiveType(PrimitiveTypeKind.Char));
```

Comment:
_(Minor)_ `PrimitiveType(PrimitiveTypeKind.Char).MakePointerType()` would be more cool.